app/workers/fetch_facebook_event_worker.rb

```diff
@@ -0,0 +1,10 @@
+class FetchFacebookEventWorker
+  include Sidekiq::Worker
+
+  def perform(event_id)
+    repository = Facebook::EventRepository.new
+    event = Facebook::Event.new(id: event_id)
+    event.load!
+    repository.cache!(event)
```

Comment:
I think that we can move `repository` assignment to another method called `repository`. The same thing whit `event`. All methods with `!`(bang) returns mandatory the boolean `true`. What do you think to move `load!` to `load` and call it into line 8? `repository.cache!(event.load)`. Maybe we can change name to `reload`.